plugins/dxp-configure/package.json

```diff
@@ -61,7 +61,7 @@
     "@flatfile/listener": "^1.0.1"
   },
   "devDependencies": {
-    "@flatfile/configure": "^0.5.39",
+    "@flatfile/configure": "^1.0.1",
```

Comment:
`devDependencies` vs `dependencies` for this one?

Reply:
Good question. This PR just updates the version. Looks like that dep has always been listed as a devDep.